app/src/test/java/org/oppia/android/app/testing/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -101,82 +86,6 @@ class AdministratorControlsFragmentTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
-  @Test
```

Comment:
It looks like all the tests are removed from this file. 

As power this issue, what I meant by shifting is, we need to only move those tests which are instrumentation test, not the Unit test. 

Check my earlier review.

Reply:
Unit tests are now available in place: https://github.com/oppia/oppia-android/pull/3185/commits/5be55859c43b45f6e243ff4a95773fdeeda62ad8